.github/workflows/ci.yml

```diff
@@ -10,30 +10,31 @@ jobs:
 
     strategy:
       matrix:
-        otp: ['25', '26', '27']
-        rebar: ['3.24']
+        otp: ['26', '27', '28']
+        rebar: ['3.25']
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: erlef/setup-beam@v1
         id: setup-beam
         with:
           otp-version: ${{matrix.otp}}
           rebar3-version: ${{matrix.rebar}}
       - name: Restore _build
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: _build
           key: _build-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
       - name: Restore rebar3's cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ~/.cache/rebar3
           key: rebar3-cache-for-os-${{runner.os}}-otp-${{steps.setup-beam.outputs.otp-version}}-rebar3-${{steps.setup-beam.outputs.rebar3-version}}-hash-${{hashFiles('rebar.lock')}}
       - name: Compile
         run: ERL_FLAGS="-enable-feature all" rebar3 compile
       #- name: Format check
-      #  run: ERL_FLAGS="-enable-feature all" rebar3 format --verify
+      #  if: matrix.otp == '28'
+      #  run: rebar3 fmt --check
```

Comment:
I was gonna try to check this from 26, but `erlfmt`'s failing there (26.2), so I'm leaving it for now, and the code was pushed formatted, and can always be reformatted, if we want.

Reply:
Humm… 🤔 … I would like to have a format check, at least in one OTP version.
I'm fine if you merge it like this and open a ticket to re-add the format check, tho.

Reply:
I've added 27 to the tests, and will stick to that, though locally I had to comment out a short line in the tests (something I imagine it doesn't parse well, yet).